CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 - .NET on iOS: Add experimental EnableAppHangTrackingV2 configuration flag to the options binding SDK ([#3877](https://github.com/getsentry/sentry-dotnet/pull/3877))
 - Added `SentryOptions.DisableSentryHttpMessageHandler`. Useful if you're using `OpenTelemetry.Instrumentation.Http` and ending up with duplicate spans. ([#3879](https://github.com/getsentry/sentry-dotnet/pull/3879))
 
+### Fixes
+
+- Use MarshalManagedExceptionMode.Default on CoreCLR ([#3912](https://github.com/getsentry/sentry-dotnet/pull/3912))
```

Comment:
is this just elegant in iOS?
if so add "iOS: bla" or macOS:iOS

Reply:
Elegant? Or did you mean relevant? 

It's relevant whenever the runtime is not mono... CoreCLR is used for AOT compiled applications on iOS so this typically affects release configuration builds of iOS apps (as long as the [interpreter](https://learn.microsoft.com/en-us/dotnet/maui/macios/interpreter?view=net-maui-8.0) isn't enabled).

I've updated the changelog to make this clearer.